src/main/java/com/example/MainTask3.java

```diff
@@ -0,0 +1,18 @@
+//3.Вывести заданное количество случайных чисел с переходом и без перехода на новую строку.
+
+package com.example;
+
+import java.util.Scanner;
+
+public class MainTask3 {
+    public static void main(String[] args){
+        Scanner enterQuantity = new Scanner(System.in);
+        System.out.println("Enter the number of random numbers: ");
+        int quantity= enterQuantity.nextInt();
+        int a =0;
+        int b = 10;
+        for (int i = 0; i < quantity; i++) {
+            System.out.println(a + (int) (Math.random()*b));
+        }
+    }
+}
```

Comment:
Please, add empty line at the and of the file.

Reply:
It's just a common practice.
Code should be consistent and homogeneous, so if you create few classes -- one with empty line at the end, another without -- it's not a clean code.

Additional links with less taste preferences, but with more information to consider:
https://stackoverflow.com/questions/4700312/empty-new-line-at-the-end-of-the-java-source-files#:~:text=newline)%2C%20or%20else-,they%20are%20trying%20to%20ensure%20every%20file%20ends%20with%20a,redundant%20additional%20newline%20%2D%20overkill%3F).
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file